react/components/DatePicker/index.js

```diff
@@ -19,6 +19,43 @@ class DatePicker extends Component {
     this.handleLocaleChange(props.locale)
   }
 
+  get popperModifiers() {
+    const { align, direction, size, useTime } = this.props
+
+    const isRightAligned = align === 'right'
+    const isUpwardsDirected = direction === 'up'
```

Comment:
`isUpwards` is enough (the suffix "ward"/"wards" already implies direction)

Reply:
It can also be used for movement, e.g. "moving upwards" or "rising upwards". I don't mind renaming it though, haha.

Reply:
so does `Directed`

Reply:
I personally prefer `direction`, but I do not oppose to renaming it 😄